schema/schema.py

```diff
@@ -0,0 +1,354 @@
+# Schema handling for test data, verification data, results files, etc.
+# For ICU Conformance project, Data Driven Testing
+import argparse
+import glob
+import json
+
+import jsonschema.exceptions
+from jsonschema import validate
+from jsonschema import ValidationError
+from jsonschema import exceptions
+
+import logging
+import os.path
+import sys
+
+import schema_files
+from schema_files import schema_file_map
+
+# ?? Move to the initialization
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+
+# Given a directory, validate JSON files against expected schema
+
+class conformance_schema_validator():
```

Comment:
Python style wants classes named using `PascalCase` (UpperCamelCase) rather than a lowercase `snake_case`. See [Google Python style guide](https://google.github.io/styleguide/pyguide.html#316-naming).

Reply:
Done